addons/editor-theme3/theme3.js

```diff
@@ -1,6 +1,28 @@
 export default async function ({ addon, global, console }) {
   var style = document.createElement("style");
-  var stylesheet = "";
+  var stylesheet = `path.blocklyBlockBackground[fill="#FF6680"],
```

Comment:
I did not fully understand what went wrong when I moved this to a userstyle. JS order is currently not guaranteed, so it's better to put all addon CSS into userstyles, where themes are guaranteed priority after normal non-theme userstyles.